src/schema-association-service.ts

```diff
@@ -93,6 +96,8 @@ async function autoDetectSchema(
     context: vscode.ExtensionContext,
     workspaceFolder: vscode.WorkspaceFolder): Promise<vscode.Uri | undefined> {
     const azureAccountApi = await getAzureAccountExtensionApi();
+    const config = vscode.workspace.getConfiguration('azure-pipelines')
+    const oneesptSchemaEnabled = vscode.workspace.getConfiguration('azure-pipelines', workspaceFolder).get<boolean>('1ESPipelineTemplatesSchemaFile', false);
```

Comment:
This variable isn't used until line 297 (I think) - please move it down there to consolidate logic.

Reply:
done